changelog/v0.17.15/less-races.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
```

Comment:
I think this is the wrong changelog content :)

Reply:
too much copy paste.. thanks!